docs/integration.rst

```diff
@@ -1110,11 +1120,16 @@ Custom bootloader backend interface
 According to :ref:`sec-boot-slot` the custom bootloader handler is called by 
 RAUC to trigger the following actions:
 
+* get the current booted slot
```

Comment:
I guess I suggested putting this at the end of the list as it is (unlike the others) more or less an optional method. So you prefer having it as the first element?

(If moving, also move the verbose description below)